### PR TITLE
added support for array of output objects.

### PR DESCRIPTION
**Pull request by mkowalski87**
Added support for array of output objects #5

### CODE REVIEW

File: Tests/XCTestParametrizedMacroTests/SimpleValuesTests.swift [hunk 1]
```diff
@@ -270,4 +270,31 @@ final class SimpleValuesTests: XCTestCase {
             macros: testMacros
         )
     }
+
+    func testParametrizeInputOutput_SingleObject() throws {
```
Comment (mmysliwiec) [inline Tests/XCTestParametrizedMacroTests/SimpleValuesTests.swift:274]: It's rather single Int so we should change the test name for clarity.

**Review by mmysliwiec — CHANGES_REQUESTED** [review]
This is a great piece of work but I think we should polish it a bit.

File: Sources/XCTestParametrizedMacroMacros/MacroDeclarationHelper.swift [hunk 2]
```diff
@@ -37,4 +54,21 @@ struct MacroDeclarationHelper {
         }
     }
 
+    var outputValues: ArrayElementListSyntax? {
+        get throws {
+            guard let firstMacroArgument = firstAttribute?.arguments?.as(LabeledExprListSyntax.self) else {
+                throw ParametrizeMacroError.macroAttributeNotAnArray
```
[the remark this answers is not in the document]
Comment (mkowalski87) [reply Sources/XCTestParametrizedMacroMacros/MacroDeclarationHelper.swift:60]: for now I've changed only text to make the message more generic

File: Tests/XCTestParametrizedMacroTests/SimpleValuesTests.swift (hunk 1 above)
Comment (mkowalski87) [reply Tests/XCTestParametrizedMacroTests/SimpleValuesTests.swift]: also removed some duplicated tests from other class.

**Review by mmysliwiec — APPROVED** [review]

**Pull request merged**